### ISSUE TITLE
[BUG]FISH demo test hang

### ISSUE THREAD

**Issue by jessehui** [occlum/occlum#244]
# Describe the bug
When using Occlum 0.18.0 to test FISH demo, the process hangs.

# To reproduce
1. Follow the readme of FISH demo to download and install all dependencies
2. execute `./run_fish_test.sh` and after occlum is running, nothing is printed out and it will hang
3. Change the first line of `fish_script.sh` to `command echo "Hello-world-from-fish" > /root/output.txt` and everything is normal
4. 

# Expected behavior
I expect running `./run_fish_test.sh` will print `Hello world from fish`.

# Logs
The last syscall log before hanging is about read:
```
[2020-11-23T09:15:40.721Z][TRACE][T3][#13][····Read] Syscall { num = Read, fd = 0, buf = 0x7fc59aa96ca0, size = 463 }
[2020-11-23T09:15:40.721Z][DEBUG][T3][#13][····Read] read: fd: 0
```
And after this, no new log shows.

# Environment

- HW: [e.g. SGX1, SGX2] SGX1
- OS: [e.g. Ubuntu18.04, CentOS8.1] Ubuntu18.04
- Occlum version: [e.g. 0.17.0] 0.18.0

# Additional context

# Possible solution/Implementation
Maybe pipe is not fully functional?

**Comment by tatetian** [comment]
Thanks for reporting the problem. I will try to reproduce the problem and find out the root cause.

**Comment by tatetian** [comment]
This bug is fixed in #254 .

**Comment by guzongmin** [comment]
fixed in 0.18.1
